app/Console/Commands/GenerateResourse.php

```diff
@@ -39,14 +39,13 @@ public function __construct()
      */
     public function handle()
     {
-        $table_name = strtolower($this->argument('controllerName'));
-        shell_exec("/usr/bin/php artisan make:controller ".$this->argument('controllerName'));
-        if(count($this->option('schema'))!= 0)
-            shell_exec("/usr/bin/php artisan make:migration:schema create_".$table_name."s_table --schema=\"".$this->option('schema')."\"");
-        else
-            shell_exec("/usr/bin/php artisan make:migration create_".$table_name."_table");
-        shell_exec("/usr/bin/php artisan make:request ".$this->argument('controllerName'));
-        shell_exec("/usr/bin/php artisan make:form Forms/".$this->argument('controllerName'));
+        $table_name = strtolower($this->argument('tableName'));
+        $form_name = $this->argument('tableName');
+        $form_name[0]=strtoupper($form_name[0]);
+        shell_exec("/usr/bin/php artisan make:controller Admin/".$this->argument('controllerName').'Controller');
```

Comment:
You can use `Artisan::call("make:controller Admin/" . $this->argument('controllerName') . "Controller")`. See the [docs](https://laravel.com/docs/5.5/artisan#programmatically-executing-commands).